sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ServiceAPITest.groovy

```diff
@@ -754,6 +754,7 @@ class ServiceAPITest extends APISpec {
         thrown(IllegalStateException)
     }
 
+    @PlaybackOnly // There is a fix in core that will be picked up with the next patch release
```

Comment:
Also need to `@PlaybackOnly` the equivalent container-level Find Blobs by Tag test

Reply:
![image](https://user-images.githubusercontent.com/61715331/166864751-480fe729-42e6-4b14-b05b-dac9409d5c96.png)